### ISSUE TITLE
Cat moves faster on diagonals

### ISSUE THREAD

**Issue by SondreDj** [crgimenes/neko#20]
Movement is not normalized leading to faster movement on diagonals.

**Comment by crgimenes** [comment]
True. Maybe if we take the dimensions of the monitor and calculate the aspect ratio and then put the proportion in a variable to have the movement closer to 1 to 1...

**Comment by sklrem** [comment]
https://www.youtube.com/shorts/0cYjreg7dpg

maybe this may give an idea to 'fix' it

**Comment by crgimenes** [comment]
> https://www.youtube.com/shorts/0cYjreg7dpg
> 
> maybe this may give an idea to 'fix' it

The problem is that I must discover a way to normalize the displacement. As the coordinates are integers and we move one pixel at a time, I need to think of a way to normalize this without rounding the number.
